applications/tari_app_grpc/proto/types.proto

```diff
@@ -68,9 +68,11 @@ message ProofOfWork {
 
 //This is used to request the which pow algo should be used with the block template
 message PowAlgo{
-    // 0 = Monero
-    // 1 = Blake
-    uint32 pow_algo = 1;
+    enum PowAlgo {
+        PowAlgoMonero = 0;
```

Comment:
```suggestion
        POW_ALGO_MONERO = 0;
```

Reply:
As per https://developers.google.com/protocol-buffers/docs/style

Reply:
`Use CamelCase (with an initial capital) for enum type names and CAPITALS_WITH_UNDERSCORES for value names`

Sounds like you're right, though I'm a little confused about the difference between a "enum type name" and "value name" - will update.